test/test_nwb_model_interpreter.py

```diff
@@ -65,10 +65,10 @@ def test_ferguson(self):
                :return:
                '''
         file_path = "/home/user/NWBShowcase/FergusonEtAl2015/FergusonEtAl2015.nwb"
-        self._single_file_test(file_path)
+        self._single_file_test(TestModelInterpreter.nwb_test_urls['ferguson2015.nwb'])
 
     def _single_file_test(self, file_path):
-        name = file_path.split('/')[-1]
+        name = os.path.basename(file_path)
```

Comment:
kudos for that! I like to get rid off of custom path splits and concatenations.

Reply:
🎉